src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -33,12 +34,14 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
     private readonly _serviceName: string = "ms-azuretools.vscode-cosmosdb.postgresPasswords";
     private _keytar: KeyTar | undefined;
     private _serverId: string;
+    private _resourceGroup: string;
```

Comment:
Just make this public - I'm sure we'll need the resourceGroup for commands outside this file as well